### PR TITLE
Fix for mobsfscan parser 2

### PR DESCRIPTION
**Pull request by hvechtomov**
Hello!
Fix for #7630. This error appeared due to the fact that in the new report, mobsfscan CWE was written in lower case.
Thank you!

### CODE REVIEW

**Comment by hvechtomov** [comment]
@Maffooch @damiencarol check this please

File: dojo/tools/mobsfscan/parser.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def get_findings(self, filename, test):
             dupes = {}
             for key, item in data.get('results').items():
                 metadata = item.get('metadata')
-                cwe = int(re.match(r'CWE-([0-9]+)', metadata.get('cwe')).group(1))
+                cwe = int(re.match(r'cwe-([0-9]+)|CWE-([0-9]+)', metadata.get('cwe')).group(1))
```
Comment (kiblik) [inline dojo/tools/mobsfscan/parser.py:35]: what about 
```python
cwe = int(re.match(r'(cwe|CWE)-([0-9]+)', metadata.get('cwe')).group(2))
```

Comment (hvechtomov) [reply dojo/tools/mobsfscan/parser.py]: changed to your suggestion

**Comment by hvechtomov** [comment]
@kiblik check please

**Comment by kiblik** [comment]
> @kiblik check please

looks good for me

**Review by Maffooch — APPROVED** [review]

**Review by devGregA — APPROVED** [review]

**Pull request merged**